k8s-install/scripts/install-cni.sh

```diff
@@ -91,9 +91,18 @@ do
     fi
     # Copy the binary to a temporary location, and then mv it into place. This safely
     # distributes the binary even if the existing executable is currently running.
-    rm -f $dir/$filename.tmp
-    cp $path $dir/$filename.tmp
-    mv $dir/$filename.tmp $dir/$filename || exit_with_error "Failed to copy $dir/$filename.tmp to $dir/$filename. This may be caused by selinux configuration on the host, or something else."
+    cp -f $path $dir/$filename.tmp
+    # Adjust security context of temporary file so the move operation below can
+    # delete temporary file without violation.
+    if [ -e "/sbin/restorecon" ]; then
```

Comment:
Do we need to worry about this binary existing in different locations on different platforms?

Reply:
Good point. I took a quick look earlier and saw many examples pointing to `/sbin`.
Now I looked at package search for rhel/centos/fedora/ubuntu etc. (e.g. https://www.rpmfind.net/linux/rpm2html/, ) and noticed `/usr/sbin` and `/sbin` as only possible locations. I have updated the PR with these 2 path details.

Let me know if there is a better way to handle it.